sp/src/game/server/ez2/npc_clonecop.cpp

```diff
@@ -67,6 +67,10 @@ CNPC_CloneCop::CNPC_CloneCop()
 	m_bThrowXenGrenades = true;
 	SetDisplacementImpossible( true );
 	m_SquadName = MAKE_STRING( IsBadCop() ? "bc_squad" : "cc_squad" );
+
+	// TODO - See comment in npc_combine.cpp
+	// Clone Cop probably sohuldn't order surrender by default
```

Comment:
So huldn't, man. So huldn't.
```suggestion
	// Clone Cop probably shouldn't order surrender by default
```